bitnami/mongodb-sharded/templates/_helpers.tpl

```diff
@@ -1,5 +1,12 @@
 {{/* vim: set filetype=mustache: */}}
 
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "mongodb-sharded.name" -}}
+{{- include "common.names.name" . -}}
```

Comment:
could we replace ocurreces of` "mongodb-sharded.name"` by `"common.names.name"` instead?

Reply:
Sure, in this regard I think `common.names.fullname` is even more appropriate as it is the preferred name in other resources.